models/semester.js

```diff
@@ -0,0 +1,52 @@
+import connector from "#models/databaseUtil";
+const semesterSchema = {
+  number:int , {enum: ["1", "2" , "3" , "4" , "5","6","7","8"], required:true},
```

Comment:
```suggestion
  number: { type: Number, enum: [1, 2, 3, 4, 5, 6, 7, 8], required: true },
```

Reply:
If the possibilities are strings, please set the type to a String, if the enum is integers then set it to Number. Please make sure you do node setup.js